google/cloud/aiplatform/models.py

```diff
@@ -14,12 +14,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import json
 import pathlib
 import proto
 import re
 import shutil
 import tempfile
-from typing import Dict, List, NamedTuple, Optional, Sequence, Tuple, Union
+
+try:
```

Comment:
This should be imported within the methods that use this module so this doesn't throw when aiplatform is imported. I think this is only the PrivateEndpoint constructors.